.github/workflows/sdk.yaml

```diff
@@ -70,8 +70,8 @@ jobs:
           cargo tarpaulin --out xml \
             --package gcp-sdk-auth \
             --package gcp-sdk-gax \
-            --package gcp-sdk-lro \
-            --package gcp-sdk-wkt
+            --package google-cloud-lro \
```

Comment:
Why not rename `gcp-sdk-gax` too?

Reply:
It exists. I want to make sure we have a plan in place with the other project first: https://crates.io/crates/google-cloud-gax

Reply:
Oh, I had missed that one. Yes, we should wait.